tests/test_crs/valid_fio_store.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: benchmark.example.com/v1alpha1
+kind: Benchmark
+metadata:
+  name: example-benchmark
+  namespace: ripsaw
+spec:
+  elasticsearch:
+    server: marquez.perf.lab.eng.rdu2.redhat.com
```

Comment:
I wonder if we should store these `valid` cr's in a private repo, due to the sensitivity of hostnames...

Reply:
Agreed, we can definitely do that. I was initially looking forward to populate the CR's sensitive data through environment variables. I'll look at it now.